lib/util/versioning.js

```diff
@@ -55,6 +69,8 @@ function get_runtime_abi(runtime, target_version) {
     }
     if (runtime === 'node-webkit') {
         return get_node_webkit_abi(runtime, target_version || process.versions['node-webkit']);
+    } else if (runtime === 'electron') {
+        return get_electron_abi(runtime, target_version || process.versions.electron);
```

Comment:
hmm. so if I'm on `electron-rebuild@1.0.1` and running it on `postinstall`, it should set it for me, eh?  curious why I still get the `Error: Empty target version is not supported if electron...` error from above